# sim_offsets.yaml
# Offsets (in degrees) for each joint, to be applied to /teachbot/joint_states_sim before publishing to /teachbot/joint_states

joint_names:
  - "shoulder_pan_joint"
  - "shoulder_lift_joint"
  - "elbow_joint"
  - "wrist_1_joint"
  - "wrist_2_joint"
  - "wrist_3_joint"
  
degree_offsets:
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  - 0.0


